.gitignore

```diff
@@ -0,0 +1,5 @@
+# nix outputs
+result*
+
+# direnv
```

Comment:
`direnv` seems to be a tool of yours? Maybe we can remove it from here for now (we probably need a better/extended project `.gitignore`, but it seems to be independent of this PR).

If you want to keep it ignored for yourself, you could write it in `.git/info/exclude`

Reply:
oh, I'm kinda used to `direnv` being a kinda ubiquitous tool to use with nix, e.g. I personally have `use flake . -Lv` in the `.envrc` file, with `nix-direnv`, this makes loading the devShell instant, it really makes working with nix much nicer :) 

but yes, I can remove it for the matter of this PR :)

Reply:
Okay interesting point. @blinry might have a different opinion than me, my exposure to nix tooling is "mainly this project".